src/OnboardingSPA/components/Button/NextButtonSiteGen/index.js

```diff
@@ -37,7 +37,8 @@ const NextButtonSiteGen = ( {
 				}
 			} }
 		>
-			{ text }
+			<p>{ text }</p>
```

Comment:
As a best practice, whenever possible, we should add a className to this p tag and target that class instead of targeting HTML elements directly.